### PR TITLE
Newsletter: Host fall newsletter browser version

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
Title^^
https://github.com/episphere/connect/issues/705

### CODE REVIEW

File: newsletter/fallnewsletter.html [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+
+			<tr>
+				<td style="background-color: #2A72A5; padding-bottom: 20px;">
+					<table width="100%">
+						<tr>
+							<td style="height: 92px; width: 559px; color: #FFFFFF; font-family: sans-serif; font-size: 18px; letter-spacing: 0; 
+							line-height: 24px; text-align: center; padding: 10px 0 20px 20px;">
+									<p>Connect for Cancer Prevention Study</p>
+									<p style="color: white">
+										<a href="https://www.cancer.gov/" style="color: white" target="_blank">National Cancer Institute</a>  |  <a href="https://www.nih.gov" style="color: white" target="_blank">National Institutes of Health<a> |
```
Comment (JoeArmani) [inline newsletter/fallnewsletter.html:330]:
```suggestion
										<a href="https://www.cancer.gov/" style="color: white" target="_blank">National Cancer Institute</a>  |  <a href="https://www.nih.gov" style="color: white" target="_blank">National Institutes of Health</a> |
```

**Review by JoeArmani — APPROVED** [review]
looks good, just one suggestion

**Review by we-ai — APPROVED** [review]
Looks good.
The image size is large (> 6 MB), making loading slow when the file is opened locally. Is it possible to reduce its size, so that our users don't need to wait for image loading (especially for slow internet)?

**Comment by abhinavjonnada82** [comment]
> Looks good. The image size is large (> 6 MB), making loading slow when the file is opened locally. Is it possible to reduce its size, so that our users don't need to wait for image loading (especially for slow internet)?

I noticed that & its a professional head shot. Ill try to reduce the image size.

**Comment by we-ai** [comment]
The image is displayed in a small box in the letter, so high resolution may not improve visual feelings much. I guess 1 MB could be sufficient. You may try different sizes and get feedbacks from ops team.

**Comment by abhinavjonnada82** [comment]
> The image is displayed in a small box in the letter, so high resolution may not improve visual feelings much. I guess 1 MB could be sufficient. You may try different sizes and get feedbacks from ops team.

Reduced head shot image to 1MB.

**Pull request merged**